server.js

```diff
@@ -16,6 +16,12 @@ const port = process.env.SERVER_PORT || 8080;
 app.use(express.static(publicDir));
 app.use(fallback(path.join(publicDir, "index.html")));
 
+app.use(
+  require("prerender-node").set("prerenderToken", [K0NREsM5STHeIKCtFnsU])
```

Comment:
paanie ale takie rzeczy to do zmiennych środowiskowych

Reply:
wygeneruj sobie nowy token a ten anuluj bo zakładam że on nie powinien być publiczny?